.github/actions/rust-build-release/src/main.py

```
@@ -290,11 +299,16 @@ def _ensure_rustup_exec() -> str:
     """Return a toolchain matching *toolchain* or its channel prefix."""
     channel_prefix = f"{toolchain}-"
+    dotted_prefix = f"{toolchain}."
     return next(
         (
             name
             for name in installed_names
-            if name == toolchain or name.startswith(channel_prefix)
+            if (
+                name == toolchain
+                or name.startswith(channel_prefix)
+                or name.startswith(dotted_prefix)
```

Ruff (0.15.10)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)